src/Pages/Signup/Signup.js

```diff
@@ -0,0 +1,260 @@
+import React, { Component } from 'react';
+import './Signup.scss';
+
+class Signup extends Component {
+  render() {
+    return (
+      <div className="Signup-Main">
+        <div className="Signup-Container">
+          <h3>회원가입</h3>
+          <p className="Signup-Sub">
+            <span className="ico">*</span>
+            필수입력사항
+          </p>
+          <main class="Signup-Form">
+            <div className="SignupWrap">
+              <div className="SignupLeft SignupId">
+                아이디
+                <span className="ico">*</span>
+              </div>
+              <div className="Signup-Input-Wrap">
+                <input
+                  type="text"
+                  name="id"
+                  maxlength="16"
+                  label="아이디"
+                  placeholder="6자 이상의 영문 혹은 영문과 숫자를 조합"
+                  data-validator="false"
+                  className="Signup-Input"
+                ></input>
```

Comment:
```suggestion
                />
```
self closing tag 적용할 수 있는 부분은 적용해주세요

Reply:
input, link, br 등 셀프 클로징 가능한 부분에 앞으로 신경쓰겠습니다~